modules/minimint-mint/src/lib.rs

```diff
@@ -348,6 +349,24 @@ impl FederationModule for Mint {
         }
     }
 
+    fn audit(&self) -> Vec<AuditItem> {
+        let mut items = vec![];
+        items.append(&mut AuditItem::from_db(
+            &self.db,
+            &ProposedPartialSignaturesKeyPrefix,
+            |v| -(v.0.amount().milli_sat as i64),
+        ));
+        items.append(&mut AuditItem::from_db(
+            &self.db,
+            &OutputOutcomeKeyPrefix,
+            |v| -(v.0.amount().milli_sat as i64),
+        ));
+        items.append(&mut AuditItem::from_db(&self.db, &NonceKeyPrefix, |v| {
+            v.milli_sat as i64
+        }));
```

Comment:
This will be a lot of items eventually, we probably need some sort of caching (e.g. a counter of issued/destroyed tokens for each amount) and a separate way to verify the cache that doesn't run every round. That might also make the DB key as part of the audit item suboptimal, although that cache might reside in the database too.

Reply:
For the issuances/spends I suppose we could just have a single aggregated record since those items should never be updated/deleted from the DB.  I believe all other modules will clean up old records?

Reply:
I think that would make for a good second commit to this PR just so we don't forget and run into this issue after a federation ran for a while. I guess one could call it a [Schlemiel the painter algorithm](https://en.wikichip.org/wiki/schlemiel_the_painter%27s_algorithm) otherwise :sweat_smile: